### ISSUE TITLE
Large note when opening 1.15 page

### ISSUE THREAD

**Issue by tmvkrpxl0** [MinecraftForge/Documentation#370]
![image](https://user-images.githubusercontent.com/41675181/114808974-458e7380-9de4-11eb-9c2e-78f7f33c6498.png)
When i open 1.15 forge documentation, note on left is huge. I have tested this with 2 browsers(Brave, FireFox) and the issue was same

**Comment by tmvkrpxl0** [comment]
I have tried light theme and issue is still same

**Comment by sciwhiz12** [comment]
This seems to be the [Version warning](https://docs.readthedocs.io/en/stable/versions.html#version-warning) feature from ReadTheDocs, where custom content can be injected into a `<main>` element when browsing a non-latest version of the documentation, as a banner to inform the reader about being on a non-latest version.

In this case, it seems that it inserts the following HTML (cleaned for readability):
```html
<div class="admonition warning">
    <p class="first admonition-title">Note</p>
    <p class="last"> You are not reading the most recent version of this documentation. <a href="/en/1.16.x/">1.16.x</a> is the latest version available.</p>
</div>
```

The problem seems to be that the admonition is a flexbox element alongside the navigation bar and page contents block, which enlargens it to match the sizes, as shown above.

Inserting this manually into the `<main class="wrapper sidebar-wrapper">` element of `base.html` (as the new first element) allows replication of the issue locally (which would otherwise be impossible because of this being a ReadTheDocs-exclusive feature). 
Because of this being a ReadTheDocs feature, ordinary PRs will be unable to affect the HTML itself; only those with actual access to the ReadTheDocs panel of the docs site can do that (and therefore fix the HTML directly). 

However, it may be possible to leverage the CSS (perhaps the `:first-child` CSS2 selector?) to probably force the admonition `<div>` to appear in the correct place. This would require either adding a new common stylesheet, or modifying the (already-minified) themed stylesheets (or manually inserting the CSS into the `base.html` template). A slight complication would be the admonition being a flexbox element, which complicates styling it to be above the navigation bar.

**Comment by tterrag1098** [comment]
I provided this CSS snippet as a fix a while back, feel free to make use of it

```css
.sidebar-wrapper > .admonition.warning {
    max-width: 200px;
    min-width: 200px;
    height: min-content;
    margin-right: 15px;
    padding-bottom: 10px;
    order: 1;
}

@media screen and (max-width: 1020px) {
    .sidebar-wrapper > .admonition.warning {
        max-width: 100%;
    }
}

.sidebar-sticky-wrapper-content {
    order: 2;
}
```

**Comment by diesieben07** [comment]
To put the warning on top full width:
```css
.sidebar-wrapper {
  flex-wrap: wrap;
}
.admonition.warning {
    width: 100%;
}
```

**Comment by tterrag1098** [comment]
That would affect all admonitions, not just this warning.

**Comment by sciwhiz12** [comment]
Fixed by #381 (commit c4247119fea67c023629e61da62f48e7d3d45a39).

> That would affect all admonitions, not just this warning.

For the record, the PR which implemented this futher narrowed the second rule to `.sidebar-wrapper > .admonition.warning`, ensuring only the version warning notice is affected by that specific rule.

**Comment by tmvkrpxl0** [comment]
> Fixed by #381 (commit [c424711](https://github.com/MinecraftForge/Documentation/commit/c4247119fea67c023629e61da62f48e7d3d45a39)).
> 
> > That would affect all admonitions, not just this warning.
> 
> For the record, the PR which implemented this futher narrowed the second rule to `.sidebar-wrapper > .admonition.warning`, ensuring only the version warning notice is affected by that specific rule.

Well, It's now fixed for 1.16 but it's not fixed for 1.15

**Comment by sciwhiz12** [comment]
@tmvkrpxl0: See the following PRs to fix the issue in all existing documentation, legacy and current (excluding 1.16 as that already has the fix): #383 for 1.12, #384 for 1.13, #385 for 1.14, #386 for 1.15, and #387 for 1.17.

**Comment by tmvkrpxl0** [comment]
> @tmvkrpxl0: See the following PRs to fix the issue in all existing documentation, legacy and current (excluding 1.16 as that already has the fix): #383 for 1.12, #384 for 1.13, #385 for 1.14, #386 for 1.15, and #387 for 1.17.

yes, it is fixed now, thanks
